lib/static/css/main.css

```diff
@@ -713,6 +713,46 @@ pre code {
   padding: 1.25rem 1.5rem;
 }
 
+/* "Copy" code block button */
+pre {
+  position: relative;
+}
+
+pre .btn-icon {
+  position: absolute;
+  top: 4px;
+  z-index: 2;
+  cursor: pointer;
+  border: 1px solid transparent;
+  padding: 0;
+  color: #b0a673;
```

Comment:
Let's make it `#fff`. It's this current color because of Babel's theme. We should opt for a neutral theme since we cater to wide variety of users.

Reply:
fixed